types/aws-lambda/trigger/dynamodb-stream.d.ts

```diff
@@ -1,20 +1,233 @@
-import { Handler } from "../handler";
```

Comment:
Leave double quotes here, no reason to change this.

Reply:
I can revert back to double quotes, but it was changed by prettier because of the config:

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/fa1941eec2a45c2f7eaed18b9042294e17e77302/.prettierrc.json#L4